jobs/config.json

```diff
@@ -11939,6 +11939,7 @@
       "--check-leaked-resources",
       "--cluster=",
       "--deployment=gke",
+      "—-down=false",
```

Comment:
looks like you got a bit of unicode in here `—-` should be `--`

Reply:
lol probably because that’s my stroke input

Reply:
oops, i copied that directly without looking closely. fixed now.